tests/pages/timeRestrictions.test.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import TimeRestrictions from '../../src/pages/timeRestrictions';
+
+describe('pages', () => {
+    describe('timeRestrictions', () => {
+        it('should render correctly', () => {
+            const tree = shallow(
+                <TimeRestrictions timeRestrictionsInfo={{ timeRestrictions: false }} csrfToken="" pageProps={[]} />,
+            );
+            expect(tree).toMatchSnapshot();
+        });
```

Comment:
We could have an additional snapshot test to check that the page correctly renders errors also